core/integration/dcs/protos/dcs.pb.go

```diff
@@ -25,7 +25,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.25.0
-// 	protoc        v3.17.3
+// 	protoc        v3.6.1
```

Comment:
Can you exclude this file as well as `ddsched.pb.go` from the PR? The line change was triggered by `make generate` but we don't really need to update this generated code as we don't touch anything near it.